fairlearn/widget/_fairlearn_dashboard.py

```diff
@@ -129,6 +131,14 @@ def __init__(
                 "model_type": ["regression"],
                 "function": r2_score_group_summary
             },
+            "f1_score": {
+                "model_type": ["classification"],
+                "function": f1_score_group_summary
+            },
+            "log_loss": {
+                "model_type": ["regression"],
```

Comment:
it should be `["probability"]`

Reply:
so in _group_metric_set.py does it go under
BINARY_CLASSIFICATION_METRICS or
REGRESSION_METRICS?

I had it under regression so far.

Reply:
this seems like a bug to me--we should have three task types, but let's leave that for a future PR. (tagging @riedgar-ms )